### PR TITLE
Dynamic ig.json file name 

### PR DESCRIPTION
**Pull request by BimBimSalaBim**
Set a dynamic name for ig.json so when changing the username it does not use the same ig.json file.

### CODE REVIEW

**Comment by b3nab** [comment]
@BimBimSalaBim please use a better variable name and follow actual code style.
`filename` doesn't express nothing. It could be all or nothing.
Rename that in another way  and stay conform to that style. You can change it to `cookie_file`.
At the moment I can't accept the PR.

**Comment by BimBimSalaBim** [comment]
@b3nab I changed the variable name to cookie_file and also fixed some things that I missed.

File: instapy_cli/cli.py [hunk 1]
```diff
@@ -77,8 +78,8 @@ def _login(self, username, password, cookie):
     def set_cookie(self, cookie):
         self.settings = cookie
     
-    def _get_ig_settings(self):
-        with open(self.settings, 'r') as igSettings:
+    def _get_ig_settings(self, filename):
```
[the remark this answers is not in the document]
Comment (b3nab) [reply instapy_cli/cli.py]: No matter for a breaking change, this is a private method you can notice from the starting underscore `_get_ig_settings`.
It's ok for me.

File: instapy_cli/cli.py [hunk 2]
```diff
@@ -19,6 +19,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         pass
     
     def _login(self, username, password, cookie):
+        filename = username + self.settings
```
[the remark this answers is not in the document]
Comment (b3nab) [reply instapy_cli/cli.py]: The username parameter is the same used for login purpose, so if you use a backslash there will be problems.. or at least your username ends with / . 
However it's fine to use a simple concatenation of strings.

[the remark this answers is not in the document]
Comment (BimBimSalaBim) [reply instapy_cli/cli.py]: Instagram only allows periods and underscores in usernames last I checked, so there should not be any problems.

**Pull request merged**